public/js/components/AtomEdit/CustomEditors/TimelineEditor.js

```diff
@@ -14,12 +15,16 @@ export class TimelineEditor extends React.Component {
   }
 
   render () {
+
     return (
       <div className="form">
         <ManagedForm data={this.props.atom} updateData={this.props.onUpdate} onFormErrorsUpdate={this.props.onFormErrorsUpdate} formName="timelineEditor">
-          <ManagedField fieldLocation="data.timeline.events" name="Events">
+          <ManagedField fieldLocation="description" name="Description" isRequired={false}>
```

Comment:
How about setting `name="Description (optional)"` instead of adding the new `fieldHelperLabel` prop?